EmployeeLeaveTracking/EmailService/EmailHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Net.Mail;
+using System.Net;
+
+namespace EmailService
+{
+    public class EmailHelper
+    {
+
+        public void SendEmail(string emailToAddress,string subject, string body)
+        {
+            using MailMessage mail = new();
```

Comment:
I would like to what is the use of this line

Reply:
SendEmail method takes three string parameters:
emailToAddress - the receiver's email address
subject - the subject of the email
body - the content or message of the email.
MailMessage mail = new(); : 
It is used to create a new object of the MailMessage class and assigns it to the variable mail. 
System.Net.Mail contains the MailMessage class and with it  an email message can be sent using the SmtpClient class.
SMTP server helps to send emails over internet.

Reply:
Alright, It seems a new way of calling a constructor of a class, now I would like to understand why are we reassigning it on line 18 
![image](https://github.com/PreetiRajguru/EmployeeLeaveTracking/assets/37995199/671adf0c-d92f-4b68-a557-e2beedee0819)

Reply:
So MailMessage and MailAddress Objects are created with different purposes : 
1.using MailMessage mail = new();
- This object automatically manages its disposal after the using block. 
- It assigns and sets up the content of the email like sender, receiver, subject, body, attachments and other properties of the email that will be sent.
- It is used to send email messages using SMTP.
2.mail.From = new MailAddress(emailFromAddress);
- Represents email address
- Used to specify sender and receiver email addresses in the MailMessage object.